dateparser/data/date_translation_data/en.py

```diff
@@ -188,7 +188,14 @@
         ],
         "in 1 decade": [
             "next decade"
-        ]
+        ],
+          "1 century ago": [
```

Comment:
the alignment for "1 century ago" is not correct, it should be moved to the left